HTML/mockup-mogo/js/renderHTML.js

```diff
@@ -0,0 +1,73 @@
+export const renderBlogs = (parent, typeOfBlog, serverData) => {
+  for (let i = 3; i < 6; i++) {
+    let child;
+
+    if (typeOfBlog === 'blog') {
+      child = `<div class="blog">
+        <picture class="blog__picture">
+            <img src=${serverData.blogs[i].previewImg} alt="office-picture">
+        </picture>
+        <div class="blog__time">
+            <h3 class="paragraph paragraph--big u-color-white">
+            ${dataConverter(serverData, 2, i)}
```

Comment:
magic numbers as 1 and 2, better to give for this function published property(as you really need only this one) and not all array of data